libs/datamodel/core/tests/attributes/composite_index.rs

```diff
@@ -376,12 +376,11 @@ fn a_bonkers_definition_2() {
     let error = datamodel::parse_schema(&dml).map(drop).unwrap_err();
 
     let expected = expect![[r#"
-        [1;91merror[0m: [1mError validating: This line is not a valid field or attribute definition.[0m
+        [1;91merror[0m: [1mError validating model "B": The index definition refers to the unknown fields:  in type A.[0m
```

Comment:
This error isn't great, but it'll automatically improve when we stop treating paths like identifiers, which we will do when we are free to improve schema_ast::ast, after we are done with rendering.